### PR TITLE
add body

### PR DESCRIPTION
**Pull request by M-Oberdiek**
is the color ok for you?

### CODE REVIEW

File: css/css-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body{
+
+    background: black;
```
Comment (DaveKrauss) [inline css/css-cheatsheet.md:3]: i like the color, nicely done

**Review by DaveKrauss — APPROVED** [review]

**Review by miriam-ertl — APPROVED** [review]
😀

File: css/css-cheatsheet.md (hunk 1 above)
Comment (Rick1Marques) [reply css/css-cheatsheet.md:3]: nice!

**Review by Rick1Marques — COMMENTED** [review]
very good

**Pull request merged**